### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by mblue9**
Have updated README here, was out of sync with vignette so https://github.com/stemangiola/tidybulk/pull/169 would help here too.

### CODE REVIEW

**Comment by mblue9** [comment]
Do you need me to change anything here to merge? As merging this would fix the old info in the readme like Bioconductor under submission

File: README.Rmd [hunk 1]
```diff
@@ -184,79 +176,23 @@ pbmc_small_polished %>%
...
-
-counts_mito = 
-	pbmc_small_polished %>%
-	
-	nest(data = -sample) %>%
-	mutate(data = map(
-		data, 
-		~ .x %>%
-			
-			# Join mitochondrion statistics
```
Comment (stemangiola) [inline README.Rmd:211]: I am a bit confused if these changes fix the figure, how come there is a bunch of code that was deleted?

Comment (mblue9) [reply README.Rmd:211]: So when I tried to render the README.Rmd to update the README.md it kept erroring (e.g. with EnsDb.Hsapiens.v86) and I saw that there was code in it that wasn't in the Introduction vignette. The README looked like it was older than the Introduction vignette so I replaced it with the Introduction code. The README code is now an exact copy of what's in the Introduction Rmd, should it be?

Comment (stemangiola) [reply README.Rmd:211]: > So when I tried to render the README.Rmd to update the README.md it kept erroring (e.g. with EnsDb.Hsapiens.v86) and I saw that there was code in it that wasn't in the Introduction vignette. The README looked like it was older than the Introduction vignette so I replaced it with the Introduction code. The README code is now an exact copy of what's in the Introduction Rmd, should it be?

Amazing!

**Pull request merged**